recipe/meta.yaml

```diff
@@ -7,12 +7,10 @@ package:
 source:
   url: https://pypi.io/packages/source/c/cx_Oracle/cx_Oracle-{{ version }}.tar.gz
   sha256: 3b2d215af4441463c97ea469b9cc307460739f89fdfa8ea222ea3518f1a424d9
-  patches:
```

Comment:
why removing this patch?  I see that patch was missing as build-dependency.  Not sure, but it might be still required